jscomp/repl.js

```diff
@@ -65,7 +65,10 @@ var cmi_files =
         `lazy`,
         `js`, `js_unsafe`, `js_re`, `js_array`, `js_null`, `js_undefined`, `js_internal`,
         `js_types`, `js_null_undefined`, `js_dict`, `js_string`, `js_vector`, 
-        `js_promise`, `js_option`, `js_float`, `js_json`
+        `js_promise`, `js_option`, `js_float`, `js_json`,
+        `arrayLabels`, `bigarray`, `bytesLabels`, `callback`, `complex`,
+        `gc`, `genlex`, `listLabels`, `moreLabels`, `oo`, `queue`, `scanf`,
+        `sort`,`stack`, `stdLabels`, `stream`, `stringLabels`
     ].map(x => `${x}.cmi:/cmis/${x}.cmi`).map(x => `--file ${x}`).join(` `)
```

Comment:
Thanks for the list, can you remove `bigarray`, `callback`, and `oo`, they don't make much sense in JS backend